cfpb/cfgov-refresh
# dbrouter shortcut function for model lookup by contenttype

### Testing
- Ensure you have the legacy db setup with a dump from content
- Clone knowledgebase app as a sibling to cfgov-refresh
 - build frontend assets `./setup.sh`
- cd back into cfgov-refresh 
 - `pip install -e ../knowledgebase`
 - `pip install simplejson`
 -  `./cfgov/manage.py update_index knowledgebase`
- Enter django admin and select a [spanish answer](http://localhost:8000/django-admin/knowledgebase/spanishanswer/1945/?_changelist_filters=p%3D9) 
- Then click `view on site` site and see then site should render
![screen shot 2016-05-03 at 10 18 04 am](https://cloud.githubusercontent.com/assets/254877/14986068/5b46d2fc-1118-11e6-86b9-913d0625da42.png)

@rosskarchner 
@kurtw
